### PR TITLE
docs: add explanatory comment before universal router stubs

### PR DESCRIPTION
**Pull request by 516310460**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/nuxt/src/app/plugins/router.ts [hunk 1]
```diff
@@ -154,7 +154,7 @@ export default defineNuxtPlugin<{ route: Route, router: Router }>((nuxtApp) => {
   const router: Router = {
     currentRoute: route,
     isReady: () => Promise.resolve(),
-    //
+    // https://router.vuejs.org/api/interfaces/routeroptions.html
```
Comment (danielroe) [inline packages/nuxt/src/app/plugins/router.ts:157]:
```suggestion
    // These options provide a similar API to vue-router but have no effect
```

Comment (516310460) [reply packages/nuxt/src/app/plugins/router.ts:157]: That's it, but why is the documentation linked here?

Comment (516310460) [reply packages/nuxt/src/app/plugins/router.ts:157]: image

Comment (516310460) [reply packages/nuxt/src/app/plugins/router.ts:157]: Thanks

**Review by danielroe — APPROVED** [review]

**Comment by pi0** [comment]
Thanks.

**Pull request merged**